extensions/amp-list/amp-list.md

```diff
@@ -197,8 +197,10 @@ The `<amp-list>` element exposes a `refresh` action that other elements can refe
 
 ### Dynamic resizing
 
+`<amp-list>` recently added support for `layout="CONTAINER"` with two caveats: (1) A list with changing contents must have a determinable initial height from a fixed-height placeholder, and (2) once content changes inside the list, resize will occur _only_ if it does not cause content jumping. This is enforced by locking the height of the list prior to rendering contents and conditionally unlocking it accordingly.
+
 [filter formats="websites, stories"]
-In several cases, we may need the `<amp-list>` to resize on user interaction. For example, when the `<amp-list>` contains an amp-accordion that a user may tap on, when the contents of the `<amp-list>` change size due to bound CSS classes, or when the number of items inside an `<amp-list>` changes due to a bound `[src]` attribute. The `changeToLayoutContainer` action handles this by changing the amp list to `layout="CONTAINER"` when triggering this action. See the following example:
+Before support for `layout="CONTAINER"` was added, in several cases, we needed the `<amp-list>` to resize on user interaction. For example, when the `<amp-list>` contains an amp-accordion that a user may tap on, when the contents of the `<amp-list>` change size due to bound CSS classes, or when the number of items inside an `<amp-list>` changes due to a bound `[src]` attribute. The `changeToLayoutContainer` action handles this by changing the amp list to `layout="CONTAINER"` when triggering this action. See the following example:
```

Comment:
I think API documentation probably always reflect the latest state; describing new features usually is done in a blog or release notes.

It may be more useful to explain when to use `layout=container` and when to use `changeToLayoutContainer` action here.

Reply:
Heh, this is tricky to explain. Let's get this in and iterate.

We can also revisit in V2. `changeToLayoutContainer` and `[is-layout-container]` probably should be changed.